spec.html

```diff
@@ -32600,11 +32600,11 @@ <h1>Atom</h1>
               1. Let _ye_ be _y_'s _endIndex_.
               1. If _direction_ is equal to +1, then
                 1. Assert: _xe_ &le; _ye_.
-                1. Let _s_ be a new List whose elements are the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
+                1. Let _s_ be a List whose elements are the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
```

Comment:
```suggestion
                1. Let _s_ be a List containing the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
```
Or perhaps better, replace all the potentially ambiguous "List containing..." text with "List whose elements are...".

Reply:
Do you think we should try to normalise that phrasing everywhere?

Reply:
Yes, I do.

Reply:
Okay, I can do that as a follow-up PR. I also prefer "whose elements are".